### ISSUE TITLE
[Feature Request] Even/Odd symmetry

### ISSUE THREAD

**Issue by meshula** [guillaumechereau/goxel#68]
Hi there! I like to build as-small-as-possible voxel models. (e.g. https://github.com/meshula/Meshes/tree/master/vox)

I love the symmetry mode in goxel, but wish there was an option for "odd" symmetry as well as even. At the moment if I am drawing with symmetry mode on, the thinnest "keel" I can make in the center of a model is two voxels wide (that's what I mean by "even" symmetry). I'd like to request a mode where the center of the model is one voxel wide, and the symmetry occurs on either side of that center slab.  aAa, vs. aAAa.

Thanks for your consideration ~

**Comment by guillaumechereau** [comment]
Yes that could be useful.  Eventually I would like to support custom symmetries, defined as a translation + rotation so that we can do all kind of things.  I guess that will be done after the refactorisation of the core I am working on at the moment.

**Comment by guillaumechereau** [comment]
By the way: you can probably achieve something similar at the moment using cloned layers.

**Comment by meshula** [comment]
I'll give that a shot.
